src/main/java/teammates/ui/webapi/CreateAccountAction.java

```diff
@@ -123,13 +126,13 @@ private String importDemoData(String instructorEmail, String instructorName, Str
         Instant now = Instant.now();
 
         // Used for start time + visible time for all sessions
-        String dateString1 = getDateString(now.minus(7, ChronoUnit.DAYS));
+        String datetimeString1 = getDatetimeString(TimeHelper.getInstantTruncatedHoursOffsetBeforeNow(2));
```

Comment:
I don't think anything inside this action needs to be changed. Sample course is not subjected to the same restriction.